contracts/OKLetsApe.sol

```diff
@@ -299,14 +329,21 @@ contract OKLetsApe is
   // Get mints left
   function getMintsLeft() public view returns (uint256) {
     uint256 currentSupply = super.totalSupply();
-    return TOTAL_TOKENS.sub(currentSupply);
+    return TOTAL_TOKENS.div(2).sub(currentSupply);
```

Comment:
This only applies if the counter type is such that minting would skip every other number, correct? Can we make this more dynamic to ensure no matter what counter type we use with the contract this would show correctly?

Reply:
031a32f